### ISSUE TITLE
In cabal files moving using 'w' skips over '.'

### ISSUE THREAD

**Issue by JonnyRa** [neovimhaskell/haskell-vim#101]
This makes it hard to edit module namespaces.

Reading https://superuser.com/questions/741422/vim-move-word-skips-dot

and then running `:verbose set iskeyword?`

gives (notice the '.' in there):
```
 iskeyword=@,48-57,_,192-255,-,.,*
        Last set from ~/.vim/plugged/haskell-vim/after/ftplugin/cabal.vim
```

Is there any reason you are doing that?

Could you change it?

I think I can probably remove the '.' using an autocmd on filetype but was wondering if that has any other consequences?

Cheers

Jonny

**Comment by raichoo** [comment]
Yep it's a bit distracting but I've did that for a reason (can't recall what exactly it was though). I should improve the cabal portion of the plugin at some point. It's still very hacky.
